### PR TITLE
My first mobile app

### PR DESCRIPTION
**Pull request by posh101**
## Best Portfolio

- Built with HTML, CSS, and GitHub
- I created a new branch
- I add HTML, CSS, and linters
- The portfolio has 11 different sections but with HTML and styled with CSS.
- I also added a readme file.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -69,7 +69,384 @@ <h4 class="sub-title">I am a Software Developer</p>
...
+    
+    </Main>
+
+  </section>
+
+  <section class="head-style">
+
+    <header class="head-h1">
+      <h1>About me</h1>
+    </header>
```
Comment (Qoosim) [inline index.html:316]: - [ ] Kindly reduce the font-size of your `h1` for it to be in line with the given template.

### Your Design

your-about-me


### Figma Design

temp-about-me

File: index.html [hunk 2]
```diff
@@ -69,7 +69,384 @@ <h4 class="sub-title">I am a Software Developer</p>
...
+    
+      <div class="program">
+
+        <ul class="prog-item22">
+          <li class="html111">html</li>
+          <li class="boot110">bootstrap</li>
+          <li class="ruby12">ruby</li>
+        </ul>
+      
+        <div class="see122">See Project</div>
```
Comment (Qoosim) [inline index.html]: - [ ] Kindly position this div and give it appropriate color like what is on the guiding template. It is not properly positioned at this moment.

button-reposition


- [ ] Kindly make the div and link or better still change it to `button` for semantic reason.

File: index.html [hunk 3]
```diff
@@ -69,7 +69,384 @@ <h4 class="sub-title">I am a Software Developer</p>
...
+        Hello I’m a software developer! <br>
+        I can help you build a product , 
+        feature or website <br>
+        Look through some of my work and <br>
+        experience! If you like what you see and <br>
+        have a project you need coded, don’t <br>
+        hestiate to contact me.
+      </p>
+
+      <div class="get">Get My Resume</div>
```
Comment (Qoosim) [inline index.html]: - [ ] Kindly make this div a link or better still a button for semantic reason.

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle: 

Hi @posh101, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

-   No linter errors. :heavy_check_mark: 
-   Correct GitHub Flow. :heavy_check_mark: 
-   Good README file. :heavy_check_mark: 



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

- [ ] Kindly replace the text `menu` on your navbar with hamburger icon as to be in line with what's on the guiding template.

### Your Design

hamburger-position


### Figma Design

temp-hamburger


- [ ] Two of your social icons on the headline part of your project is not showing correct. You may need to consider checking what's the issue.

social-icons

**Review by Owusu-Desmond — APPROVED** [review]
Hi @posh101  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the set up the project ✅
📌 Good commit messages.

## Optional suggestions

- [ OPTIONAL ] Your readme file is not professional. Please would be better to make your readme file professional since it is the first impression if someone visit to project here. Please make sure you follow [Microverse README template](https://github.com/microverseinc/readme-template/blob/master/README.md) :pray: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by posh101** [comment]
Hello

Please i have finished my quiz and submitted but it has not reflected on my
dashboard and I have not seen my scores.

**Pull request merged**